examples/kubernetes/cross_account_mount/README.md

```diff
@@ -0,0 +1,58 @@
+## Dynamic Provisioning
+This example shows how to create a dynamically provisioned volume created through [EFS access points](https://docs.aws.amazon.com/efs/latest/ug/efs-access-points.html) and Persistent Volume Claim (PVC) and consume it from a pod.
+
+**Note**: this example requires Kubernetes v1.17+ and driver version >= 1.2.0.
+
+### Edit [StorageClass](./specs/storageclass.yaml)
+
+```
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: efs-sc
+provisioner: efs.csi.aws.com
+mountOptions:
+  - tls
+parameters:
+  provisioningMode: efs-ap
+  fileSystemId: fs-1234abcd
+  directoryPerms: "700"
+  gidRangeStart: "1000"
+  gidRangeEnd: "2000"
+  basePath: "/dynamic_provisioning"
+  az: us-east-1
+```
+* provisioningMode - The type of volume to be provisioned by efs. Currently, only access point based provisioning is supported `efs-ap`.
+* fileSystemId - The file system under which Access Point is created.
+* directoryPerms - Directory Permissions of the root directory created by Access Point.
+* gidRangeStart (Optional) - Starting range of Posix Group ID to be applied onto the root directory of the access point. Default value is 50000. 
+* gidRangeEnd (Optional) - Ending range of Posix Group ID. Default value is 7000000.
+* basePath (Optional) - Path on the file system under which access point root directory is created. If path is not provided, access points root directory are created under the root of the file system.
+* `az` under storage class parameter is optional. If specified, mount target associated with the az will be used for cross-account mount. If not specified, a random mount target will be picked for cross account mount
+
+### Prerequisite setup
+1.  Perform [vpc-peering](https://docs.aws.amazon.com/vpc/latest/peering/working-with-vpc-peering.html) between EKS cluster `vpc` and EFS `vpc`.
```

Comment:
the cluster is in account A and EFs in B? or other way around

Reply:
Cluster is in account A and efs in account B. Let me update it here for clarity.